lib/diff_web/controllers/page_controller.ex

```diff
@@ -36,14 +36,29 @@ defmodule DiffWeb.PageController do
     end
   end
 
-  defp parse_versions(versions) do
-    with [from, to] <- String.split(versions, "..", trim: true),
-         {:ok, from} <- Version.parse(from),
-         {:ok, to} <- Version.parse(to) do
+  defp parse_versions(package, versions) do
+    with [from, to] <- versions |> String.split("..") |> Enum.map(&String.trim/1),
+         {:ok, from} <- parse_version(package, :from, from),
+         {:ok, to} <- parse_version(package, :to, to) do
```

Comment:
What happens `:from` is already the latest version?

Reply:
Good point! After this other commit https://github.com/hexpm/diff/pull/45/commits/b9071e34c235a14886d5f9aa03c0bf3ab8a63e0f , it is 400'ing when from is empty (e.g. "/diff/package/%20.."), and also when the versions are the same. Would you want a 400 in this last case?